Scripts/black_temple_cardcommentgen.py

```diff
@@ -0,0 +1,231 @@
p","mechanics":["TRIGGER_VISUAL"],"name":"Parry and Riposte","set":"BLACK_TEMPLE","text":"<b>Passive Hero Power</b>\nThe first time you are damaged each turn, prevent it and counterattack!","type":"HERO_POWER"},{"cardClass":"NEUTRAL","dbfId":62514,"health":60,"id":"BTA_BOSS_24h","name":"Lady Vashj","set":"BLACK_TEMPLE","type":"HERO"},{"cardClass":"SHAMAN","cost":1,"dbfId":62618,"id":"BTA_BOSS_24p","name":"Shock Blast","set":"BLACK_TEMPLE","text":"[x]<b>Hero Power</b>\nDeal $3 damage to an\nenemy minion. If it\nsurvives, draw a card.","type":"HERO_POWER"},{"attack":6,"cardClass":"SHAMAN","cost":6,"dbfId":62619,"health":8,"id":"BTA_BOSS_24t","mechanics":["RUSH","TRIGGER_VISUAL"],"name":"Coilfang Elite","set":"BLACK_TEMPLE","text":"<b>Rush</b>\nAfter this attacks, summon a Neutral Murloc.","type":"MINION"},{"cardClass":"NEUTRAL","dbfId":62517,"health":60,"id":"BTA_BOSS_25h","name":"Kael'thas Sunstrider","set":"BLACK_TEMPLE","type":"HERO"},{"cardClass":"WARLOCK","cost":0,"dbfId":62620,"hideStats":True,"id":"BTA_BOSS_25p","name":"Pyromancy","set":"BLACK_TEMPLE","text":"[x]<b>Passive Hero Power</b>\n  After you cast a spell,\nadd a Pyroblast to your\n deck. It costs (5) less.","type":"HERO_POWER"},{"cardClass":"NEUTRAL","cost":2,"dbfId":62902,"hideStats":True,"id":"BTA_BOSS_25p2","name":"Gravity Lapse","set":"BLACK_TEMPLE","text":"<b>Passive Hero Power</b>\n You can't be attacked.","type":"HERO_POWER"},{"dbfId":62621,"id":"BTA_BOSS_25pe","name":"Pyromancy Enchantment","set":"BLACK_TEMPLE","text":"Costs (5) less.","type":"ENCHANTMENT"},{"cardClass":"NEUTRAL","cost":2,"dbfId":62622,"id":"BTA_BOSS_25s","name":"Gravity Lapse","set":"BLACK_TEMPLE","text":"[x]Return all enemy minions\nto your opponent's hand.\nThey cost (3) more.","type":"SPELL"},{"cardClass":"NEUTRAL","dbfId":62918,"id":"BTA_BOSS_25se","name":"Gravity Lapse Enchantment","set":"BLACK_TEMPLE","text":"Costs (3) more.","type":"ENCHANTMENT"},{"cardClass":"DEMONHUNTER","dbfId":62519,"health":60,"id":"BTA_BOSS_26h","name":"Illidan Stormrage","set":"BLACK_TEMPLE","type":"HERO"},{"cardClass":"WARLOCK","cost":0,"dbfId":62623,"hideStats":True,"id":"BTA_BOSS_26p","mechanics":["TRIGGER_VISUAL"],"name":"Burning Rage","set":"BLACK_TEMPLE","text":"[x]<b>Passive Hero Power</b>\nAfter your hero attacks,\ndeal $1 damage to all\nenemies.","type":"HERO_POWER"},{"cardClass":"DEMONHUNTER","cost":4,"dbfId":62624,"id":"BTA_BOSS_26s","name":"You Are Not Prepared","set":"BLACK_TEMPLE","text":"[x]Give your hero +2 Attack\nand <b>Lifesteal</b> this turn, \nthen attack all enemy\ncharacters.","type":"SPELL"},{"cardClass":"DEMONHUNTER","dbfId":62625,"id":"BTA_BOSS_26se","mechanics":["TAG_ONE_TURN_EFFECT"],"name":"Prepared","set":"BLACK_TEMPLE","text":"Has +2 Attack and <b>Lifesteal</b> this turn.","type":"ENCHANTMENT"},{"cardClass":"NEUTRAL","dbfId":62122,"id":"BTA_Prevent_First_turn_Attack","mechanics":["TAG_ONE_TURN_EFFECT"],"name":"Exhausted","set":"BLACK_TEMPLE","text":"Can't attack this turn.","type":"ENCHANTMENT"}]
```

Comment:
I don't recommend this way. Can you read the data from 'cards.json' and filter specific card set?